lib/misc.c

```diff
@@ -162,6 +162,13 @@ void strip_html(char *in)
 
 	memset(out, 0, sizeof(out));
 
+	if (strlen(in) == 0) {
```

Comment:
You could do `if (!*in)` here so that it's equivalent to the check below - calling strlen isn't really needed anyway.